src/utils/trust/legal.entity.beneficial.owner.mapper.ts

```diff
@@ -123,18 +142,28 @@ const mapLegalEntityTrusteeFromSessionToPage = (
     registration_number: trustee.identification_registration_number,
     is_service_address_same_as_principal_address: trustee.is_service_address_same_as_principal_address,
     is_on_register_in_country_formed_in: trustee.is_on_register_in_country_formed_in,
+    ceasedDateDay: trustee.ceased_date_day,
+    ceasedDateMonth: trustee.ceased_date_month,
+    ceasedDateYear: trustee.ceased_date_year,
   } as Page.TrustLegalEntityForm;
 
-  if (trustee.type === RoleWithinTrustType.INTERESTED_PERSON){
+  const stillInvolvedInTrust: string =
```

Comment:
Add tests for the case "yes" and "No" but also "Wrong answer" and undefined

Reply:
Test added